gpg/encrypt/encrypt_test.go

```diff
@@ -21,8 +21,7 @@ func Test_GPG_Encrypt_Service_Modify(t *testing.T) {
 
 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
-xxxxxxxxxxxxxxxxxxxxxxxxxxxx
-xxxxx
+xxxxxxxxxxxxxx
```

Comment:
Changes in the crypto libs changed the encryption which makes the messages shorter. Note that decryption keeps being compatible.